packages/cascara/src/modules/ModuleErrorBoundary/ModuleErrorBoundary.js

```diff
@@ -10,9 +10,7 @@ const propTypes = {
 
 const ModuleErrorBoundary = ({ children }) => {
   return (
-    <Boundaries ErrorFallbackComponent={ModuleErrorFallback}>
-      {children}
-    </Boundaries>
+    <Boundaries ErrorComponent={ModuleErrorFallback}>{children}</Boundaries>
```

Comment:
@lordpixel I did a quick rename once I added support for Suspense in our Boundaries component so we would not have overly complex names.